web/src/app/administration/components/import-export/import-export.component.ts

```diff
@@ -127,19 +127,12 @@ export class ImportExportComponent implements OnInit {
     }
   }
 
-  private onFailedResponse(event) {
-    // new Key ERROR_TYPES.UPLOAD_ERR
-    this.errorHandler('Upload failed', 'The upload didn\'t proceed sucessfully. \n'
-    + 'Probably the uploaded file exceeded the maximum file size of 10 MB');
+  private onFailedResponse() {
+    this.errorHandler(ERROR_TYPES.UPLOAD_ERR);
   }
 
-  private errorHandler(title = 'Import was not successful', message) {
-    this.generalModalService.triggerMessage(
-      new MessageModal(
-        title,
-        message
-      )
-    );
+  private errorHandler(key: ERROR_TYPES, passedError?: HttpErrorResponse) {
+    this.errorsService.updateError(key, passedError);
```

Comment:
Is this errorHandler necessary? We could call errorsService directly to be consistent. In other components, there is no errorHandler.